### PR TITLE
liquibase configuration + address migration script (without recovering

### PR DESCRIPTION
**Pull request by simionescuraluca**
old address data)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,52 +20,58 @@
...
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-maven-plugin</artifactId>
+			<version>3.4.1</version>
+		</dependency>
 		<dependency>
 			<groupId>org.modelmapper</groupId>
 			<artifactId>modelmapper</artifactId>
 			<version>2.3.2</version>
 		</dependency>
-
```
Comment (dragosslujeru) [inline pom.xml]: revert formatting changes in pom file, leave only liquibase dependency addition

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,4 +2,6 @@ spring.jpa.hibernate.ddl-auto=none
 spring.datasource.url=jdbc:mysql://localhost:3306/vinylstore
 spring.datasource.username=root
 spring.datasource.password=Za2s3Tcvas
-spring.jpa.database = MYSQL
\ No newline at end of file
+spring.jpa.database = MYSQL
+#spring.liquibase.change-log=classpath:/db/changelog-profiles/changelog-master.xml
```
Comment (dragosslujeru) [inline src/main/resources/application.properties]: remove commented line

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,4 +2,6 @@ spring.jpa.hibernate.ddl-auto=none
...
+spring.liquibase.change-log=classpath:changelogOrg.xml
```
Comment (dragosslujeru) [inline src/main/resources/application.properties]: rename changelog file to have a more significant name "changelog-all" maybe

File: src/main/resources/changelogs/0001-address-migration.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+	<changeSet id="1_create_address_table" author="raluca">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="ADDRESS" schemaName="vinylstore" />
+			</not>
+		</preConditions>
+		<createTable tableName="ADDRESS" schemaName="vinylstore">
+			<column name="id" type="int">
+				<constraints primaryKey="true" />
```
Comment (dragosslujeru) [inline src/main/resources/changelogs/0001-address-migration.xml:17]: autoincrement

File: src/main/resources/application.properties (hunk 2 above)
Comment (simionescuraluca) [reply src/main/resources/application.properties]: done

File: src/main/resources/changelogs/0001-address-migration.xml (hunk 4 above)
Comment (simionescuraluca) [reply src/main/resources/changelogs/0001-address-migration.xml:17]: it s there:))

File: src/main/resources/changelogs/0001-address-migration.xml [hunk 5]
```diff
@@ -37,11 +63,48 @@
 			</not>
 		</preConditions>
 		<addColumn schemaName="vinylstore" tableName="USER">
-			<column name="address_id" type="int" />
+			<column name="address_id" type="int">
```
Comment (dragosslujeru) [inline src/main/resources/changelogs/0001-address-migration.xml:66]: <column name="address_id" type="int" /> if this style works

File: src/main/java/com/vinyl/service/UserService.java [hunk 6]
```diff
@@ -32,9 +37,12 @@ public User addUser(User user) {
 
 		Role r = new Role("BASIC_USER");
 		r = roleRepository.save(r);
-		user.setAddress("Defaul Address");
 		user.setRole(r);
 
+
+		Address a = addressRepository.findByCountryAndCityAndStreetAndNumber("Romania", "Iasi", "Strada Palat", 1);
```
Comment (dragosslujeru) [inline src/main/java/com/vinyl/service/UserService.java]: change name to of variable defaultAddress

**Review by dragosslujeru — APPROVED** [review]

**Pull request merged**